api-put-object-streaming.go

```diff
@@ -229,12 +219,12 @@ func (c *Client) putObjectMultipartStreamFromReadAt(ctx context.Context, bucketN
 			return UploadInfo{}, ctx.Err()
 		case uploadRes := <-uploadedPartsCh:
 			if uploadRes.Error != nil {
+
 				return UploadInfo{}, uploadRes.Error
 			}
 
 			// Update the totalUploadedSize.
 			totalUploadedSize += uploadRes.Size
-			// Store the parts to be completed in order.
```

Comment:
Keep the comments @bwplotka 

```
// Store the parts to be completed in upload order.
```

Reply:
Yea, this comment was quite confusing we don't store things in order, we only sort it later. Happy to make it clear in this comment. 🤗